pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -483,3 +488,30 @@ func (k *Bootstrapper) applyKicOverlay(cfg config.MachineConfig) error {
 	}
 	return nil
 }
+
+// applyNodeLabels applies minikube labels to all the nodes
+func (k *Bootstrapper) applyNodeLabels(cfg config.MachineConfig) error {
+	start := time.Now()
+	// based on ISO 8601 (RFC 3339) except converting - and : to _ because of kubernetes label restriction
+	createdAtLbl := "k8s.minikube.io/updated_at=" + start.Format("2006_01_02T15_04_05_0700")
+	verLbl := "k8s.minikube.io/version=" + version.GetVersion()
+	commitLbl := "k8s.minikube.io/commit=" + version.GetGitCommitID()
```

Comment:
Can we omit the commit id? At least until someone declares a use case for it.

Reply:
I think the use case for it is, if I am developing locally, I wanna know my developoing-code created this cluster not the minikube stable